.github/workflows/compress-pdfs.yml

```
@@ -0,0 +1,85 @@
+        else
+          echo "changed=false" >> $GITHUB_OUTPUT
+        fi
+    
+    - name: Commit compressed PDFs
+      if: steps.verify-changed-files.outputs.changed == 'true'
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        git add slides/
+        git commit -m "chore: compress PDF slides to reduce repository size [skip ci]"
+        git push
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)